storybook/stubs/shared/stores/send/TransactionStore.qml

```diff
@@ -53,15 +58,22 @@ QtObject {
                            address: "0x2B748A02e06B159C7C3E98F5064577B96E55A7b4",
                            chainShortNames: "eth:arb1"
                        })
```

Comment:
it would be nice to add {} for the for-loop statement

Reply:
Yeah, not my code :shrug: